db/migrate/20160106040041_alter_loans.rb

```diff
@@ -0,0 +1,12 @@
+class AlterLoans < ActiveRecord::Migration
+  def change
+    add_column :loans, :name, :string
```

Comment:
Odd.  I see now that it already exists.  Not sure why I thought it  was missing before.  It didn't seem to be fatal on my system, but I'll rip it out and make sure things are still working.